io.mrarm.mcpelauncher-ui.json

```diff
@@ -0,0 +1,128 @@
+{
+    "id": "io.mrarm.mcpelauncher-ui",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.14",
+    "command": "mcpelauncher-ui-qt",
+    "build-options" : {
+        "cflags": "-O2 -g -DNDEBUG",
+        "cxxflags": "-O2 -g -DNDEBUG"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/bin/protoc",
+        "/bin/zipcmp",
+        "/bin/zipmerge",
+        "/bin/ziptool",
+        "/lib/libprotobuf-lite*",
+        "/lib/libprotoc*"
+    ],
+    "finish-args": [
+        "--socket=x11",
+        "--share=network",
+        "--filesystem=host",
```

Comment:
Why does it need full `host` access?

Reply:
The file picker don't need it, removed now.